docs/content/en/content-management/urls.md

```diff
@@ -89,67 +89,149 @@ If you set both `slug` and `url` in front matter, the `url` value takes preceden
 
 ### Permalinks
 
-In your site configuration, set a URL pattern for regular pages within a top-level section. This is recursive, affecting descendant regular pages.
+In your site configuration, define a URL pattern for each top-level section. Each URL pattern can target a given language and/or [page kind].
 
-{{% note %}}
-The  `permalinks` defined in your site configuration do not apply to section pages. To adjust the URL for section pages, set `url` in front matter.
-{{% /note %}}
+Front matter `url` values override the URL patterns defined in the `permalinks` section of your site configuration.
 
-#### Examples {#permalinks-examples}
+[page kind]: https://gohugo.io/templates/section-templates/#page-kinds
+
+#### Monolingual examples {#permalinks-monolingual-examples}
 
 With this content structure:
 
 ```text
 content/
 ├── posts/
-│   ├── _index.md
-│   ├── post-1.md
-│   └── post-2.md
+│   ├── bash-in-slow-motion.md
+│   └── tls-in-a-nutshell.md
+├── tutorials/
+│   ├── git-for-beginners.md
+│   └── javascript-bundling-with-hugo.md
 └── _index.md
 ```
 
-Create a date-based hierarchy, recursively, for regular pages within the `posts` section:
+Render tutorials under "training", and render the posts under "articles" with a date-base hierarchy:
 
 {{< code-toggle file="hugo" copy=false >}}
-[permalinks]
-  posts = '/posts/:year/:month/:title/'
+[permalinks.page]
+posts = '/articles/:year/:month/:slug/'
+tutorials = '/training/:slug/'
+[permalinks.section]
+posts = '/articles/'
+tutorials = '/training/'
```

Comment:
To ne, the above will only make sense in the simple case (one level of sections). Perhaps something ala:

```
[permalinks.section]
posts = '/articles/:sections[1:]'
tutorials = '/training/:sections[1:]'
```